projects/sdk/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@beanstalk/sdk",
   "version": "0.1.0-beta.9",
-  "description": "Beanstalk protocol SDK",
+  "description": "Beanstalk protocol SDK project",
```

Comment:
any reason for this?

Reply:
It's a generic change to trigger the workflow. We can revert this before merging if you want.

Reply:
Updated in my commit with metadata above